WeightProducer/python/MCSampleValues.py

```diff
@@ -935,62 +763,52 @@ class MCSampleValuesHelper():
+                 XS_13TeV=18540.0, XSSource_13TeV="XSDB (LO)",
+            ),
+        },
         "GJets_HT-100To200" : {
             "CrossSection" : XSValues(
-                XS_2016=9249.0, XSSource_2016="XSDB (LO)",
-                XS_2017=8640.0, XSSource_2017="XSDB (LO)",
-                XS_2018=8608.0, XSSource_2018="GenXSecAnalyzer",
+                XS_13TeV=8644.0, XSSource_13TeV="XSDB (LO)",
             ),
         },
         "GJets_HT-200To400" : {
             "CrossSection" : XSValues(
-                XS_2016=2321.0, XSSource_2016="XSDB (LO)",
-                XS_2017=2185.0, XSSource_2017="XSDB (LO)",
-                XS_2018=2190.0, XSSource_2018="GenXSecAnalyzer",
+                XS_13TeV=2183.0, XSSource_13TeV="XSDB (LO)",
             ),
         },
         "GJets_HT-400To600" : {
             "CrossSection" : XSValues(
-                XS_2016=275.2, XSSource_2016="XSDB (LO)",
-                XS_2017=258.5, XSSource_2017="GenXSecAnalyzer",
-                XS_2018=258.0, XSSource_2018="GenXSecAnalyzer",
+                XS_13TeV=260.2, XSSource_13TeV="GenXSecAnalyzer",
             ),
         },
         "GJets_HT-600ToInf" : {
             "CrossSection" : XSValues(
-                XS_2016=93.19, XSSource_2016="XSDB (LO)",
-                XS_2017=85.31, XSSource_2017="XSDB (LO)",
-                XS_2018=85.11, XSSource_2018="GenXSecAnalyzer",
+                XS_13TeV=86.29, XSSource_13TeV="XSDB (LO)",
             ),
         },
         "GJets_DR-0p4_HT-100To200" : {
             "CrossSection" : XSValues(
-                XS_2016=5363.0, XSSource_2016="XSDB (LO)",
-                XS_2017=5044.0, XSSource_2017="GenXSecAnalyzer",
-                XS_2018=5030.0, XSSource_2018="GenXSecAnalyzer",
+                XS_13TeV=5030.0, XSSource_13TeV="GenXSecAnalyzer",
             ),
         },
         "GJets_DR-0p4_HT-200To400" : {
             "CrossSection" : XSValues(
-                XS_2016=1178.0, XSSource_2016="XSDB (LO)",
-                XS_2017=1130.0, XSSource_2017="GenXSecAnalyzer",
-                XS_2018=1125.0, XSSource_2018="GenXSecAnalyzer",
+                XS_13TeV=1128.0, XSSource_13TeV="GenXSecAnalyzer",
             ),
         },
         "GJets_DR-0p4_HT-400To600" : {
             "CrossSection" : XSValues(
-                XS_2016=131.8, XSSource_2016="XSDB (LO)",
-                XS_2017=124.6, XSSource_2017="GenXSecAnalyzer",
-                XS_2018=124.7, XSSource_2018="GenXSecAnalyzer",
+                XS_13TeV=126.7, XSSource_13TeV="GenXSecAnalyzer",
             ),
         },
         "GJets_DR-0p4_HT-600ToInf" : {
             "CrossSection" : XSValues(
-                XS_2016=44.27, XSSource_2016="XSDB (LO)",
-                XS_2017=40.65, XSSource_2017="GenXSecAnalyzer",
-                XS_2018=40.46, XSSource_2018="GenXSecAnalyzer",
+                XS_13TeV=41.39, XSSource_13TeV="GenXSecAnalyzer",
             ),
         },
+        #TODO
```

Comment:
Please remove this comment and all of the other `#TODO` comments.

Reply:
or clarify *what* to do

Reply:
Cleaned up, keeping track in the new issue todo list.